datajoint/fetch.py

```diff
@@ -170,6 +171,18 @@ def __call__(
         :param download_path: for fetches that download data, e.g. attachments
         :return: the contents of the relation in the form of a structured numpy.array or a dict list
         """
+        if disable_checksum and config["safemode"]:
```

Comment:
Do you mean raise a warning on fetches that they explicitly disable the checksum? That makes sense to me. The logs might just get a bit verbose in the output... This would eliminate the need for interacting with `safemode` in this option.

Unless you mean raise a warning like: `WARNING: Checksum evaluation enabled for fetches from external storage`? If so, I think that would defeat the purpose as users would be motivated to disable the checksum just to eliminate the warning.

Reply:
Overall, I think that setting this in every fetch command is not the right approach. This should be set globally. Warnings can be generated not at the time of issuing the fetch but at the time when a blob completes downloading without a checksum check.